src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.vb

```diff
@@ -122,6 +124,11 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
 
         Private Sub InitializeIconFile()
             GetProjectsAndProvider()
+            Dim PackageIconUrlSet = TryCast(TryGetNonCommonPropertyValue(GetPropertyDescriptor(_packageIconUrlPropName)), String)
+            If (PackageIconUrlSet IsNot Nothing AndAlso PackageIconUrlSet IsNot "") Then
+                SetPackageIconUrlWarninglWarningActive(True)
```

Comment:
Nothing ever calls this with false, so either the parameter is unnecessary, or there is a bit of code missing.

Reply:
Not sure where to add the call with false value, looking after that

Reply:
Added this! :)